evalai/views.py

```diff
@@ -0,0 +1,22 @@
+from django.shortcuts import render_to_response
+from django.template import RequestContext
+
+# HTTP Error 404
```

Comment:
please delete this line and make sure there is a gap of two lines only before starting the function definition?